### PR TITLE
feat(rviz_plugin): console meter is too large on the Rviz with FHD display, isn't it?

### PR DESCRIPTION
**Pull request by TakaHoribe**
Note: must be merged with https://github.com/autowarefoundation/autoware_launch/pull/44

## Description

I wanted to adjust the size of the plugins to our screen size.

I modified the default size so that these plugins are displayed at smaller sizes on a screen smaller than 4k.
Note: I'm not satisfied with my approach. (Although I hate the current size more.) Any better solution is welcome. 

Now on FHD: Steering, you are not the main character.
Screenshot from 2022-03-25 19-02-47

With this PR. It's cool. Nice.

on 4k.
Screenshot from 2022-03-25 19-02-03

on FHD.
Screenshot from 2022-03-25 19-00-51

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: common/tier4_vehicle_rviz_plugin/src/tools/turn_signal.cpp [hunk 1]
```diff
@@ -18,22 +18,32 @@
...
 {
 TurnSignalDisplay::TurnSignalDisplay()
 {
+  const Screen * screen_info = DefaultScreenOfDisplay(XOpenDisplay(NULL));
+
+  const bool large_screen = screen_info->height > 2000;
+  const auto left = large_screen ? 196 : 98;
+  const auto top = large_screen ? 350 : 175;
+  const auto width = large_screen ? 512 : 256;
+  const auto height = large_screen ? 256 : 128;
```
Comment (yukkysaito) [inline common/tier4_vehicle_rviz_plugin/src/tools/turn_signal.cpp:33]: Rather than switching between FHD or 4K, it would be better to fix the ratio.

**Comment by wep21** [comment]
@TakaHoribe Are you still working on this PR?

**Comment by TakaHoribe** [comment]
@wep21 I didn't touch this PR recently but will make it. I would be appreciated if someone could help.

**Comment by TakaHoribe** [comment]
now i'm taking it

File: common/tier4_vehicle_rviz_plugin/src/tools/turn_signal.cpp (hunk 1 above)
Comment (TakaHoribe) [reply common/tier4_vehicle_rviz_plugin/src/tools/turn_signal.cpp]: addressed in [6dc12a9](https://github.com/autowarefoundation/autoware.universe/pull/587/commits/6dc12a92b61fdcbe6ae881c752ff6e0977675460)

**Comment by TakaHoribe** [comment]
@wep21 @yukkysaito now it is ready for review

**Comment by TakaHoribe** [comment]
FHD
Screenshot from 2022-05-24 13-23-45


4K
Screenshot from 2022-05-24 13-21-29

**Comment by yukkysaito** [comment]
LGTM
Screenshot from 2022-05-26 01-40-14

**Pull request merged**